.pre-commit-config.yaml

```diff
@@ -60,7 +60,7 @@ repos:
   hooks:
   - id: mypy
     exclude: tests
-    args: ["--pretty"]
+    args: ["--pretty", "--no-implicit-optional"]
```

Comment:
Thoughts on moving this to `setup.cfg` with the rest of the mypy configuration?

https://github.com/pypa/pip/blob/89326c8a9f91e533b6cae693340fc9ae06e1df5d/setup.cfg#L33-L37

Reply:
Hmm... Maybe we can do this.

Reply:
We should probably add --show-error-codes (whatever the exact spelling is) here.

Reply:
Though, that can probably done separately from this PR.

Reply:
Looks like that was already done in 2aab462b7f30de804aefefceb5768f37ff0b5131.

Reply:
Yeah, the `--error-codes` flag is already there.